index.js

```diff
@@ -121,7 +121,10 @@ class DockerExecutor extends Executor {
      * @return {Promise}
      */
     _start(config) {
-        const [buildImage, buildTag] = config.container.split(':');
+        const containerName = config.container.split(':');
+
+        const buildTag = (containerName.length === 1) ? 'latest' : containerName.pop();
```

Comment:
Since you are checking the `buildTag` here, this line can just be `tag: buildTag`
https://github.com/screwdriver-cd/executor-docker/blob/ad8d2792617a2ce746233a02ce74035ec49cb667/index.js#L137